nvram.h

```diff
@@ -1,79 +1,118 @@
 #ifndef INCLUDE_NVRAM_H
 #define INCLUDE_NVRAM_H
 
+// We prefix every "exposed" function with `libinject_` as we don't want to clobber
+// anything by default.
+// Internal methods will be prefixed with `_libinject_`.
+
 // Locks the nvram directory. Will block.
-static int dir_lock();
+static int _libinject_dir_lock();
 // Unlocks the nvram directory.
-static void dir_unlock(int dirfd);
+static void _libinject_dir_unlock(int dirfd);
```

Comment:
I know it was like this before, but we should maybe not have the internal functions in the header file, since we want to include it from other projects. We can just put these prototypes in nvram.c I think.

Reply:
I moved dir_unlock/dir_lock into nvram.c. Kept the methods in `strings.h` since they're shared by `nvram.c` and `strings.c`